src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAXIMUM_NUMBER = 100;
+
+    public Lottery() {
+    }
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), new Random().nextInt(MAXIMUM_NUMBER));
```

Comment:
look at my previous  comments  about initializations ColorSupplier and Random

Reply:
Fixed it, thank you